packages/core/src/database/sqlite/service.ts

```diff
@@ -498,6 +498,42 @@ export class SqliteDatabaseService implements BaseDatabaseService {
     });
   }
 
+  async addIndexes({ schema }: { schema: Schema }) {
+    await Promise.all(
+      Object.entries(getTables(schema)).flatMap(([tableName, table]) => {
+        if (table[1] === undefined) return [];
+
+        return Object.entries(table[1]).map(async ([name, index]) => {
+          await this.db.wrap({ method: "addIndexes" }, async () => {
+            const indexName = `${tableName}_${name}`;
+
+            const indexColumn = index[" column"];
+            const columns = Array.isArray(indexColumn)
+              ? indexColumn.map((ic) => `"${ic}"`).join(", ")
+              : `"${indexColumn}"`;
+
+            await this.db.executeQuery(
+              sql`CREATE INDEX "${sql.raw(this.userNamespace)}"."${sql.raw(
+                indexName,
+              )}" ON "${sql.raw(tableName)}" (${sql.raw(columns)})`.compile(
+                this.db,
+              ),
```

Comment:
Let's be careful to use the correct `sql` functions here to ensure that we're escaping user input.

https://kysely-org.github.io/kysely-apidoc/interfaces/Sql.html